src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -264,14 +265,24 @@ public static void UpdateBackgroundLayerFrame(this UIView view)
 
 			var layer = view.Layer;
 
+			UpdateBackgroundLayerFrame(layer, view.Bounds);
```

Comment:
Not in 100% of the cases we will have the background layer as a sublayer of the main one. The main change is here, use recursion to get the background layer.